### ISSUE TITLE
Add Cartridge Controller

### ISSUE THREAD

**Issue by ponderingdemocritus** [apibara/starknet-react#517]
Task to add Cartridge Controller to the default connectors.

**Comment by Iwueseiter** [comment]
Hi @ponderingdemocritus can I work on this?

**Comment by fracek** [comment]
That's a great idea! BTW someone from Cartridge should get in touch because I'm going to revamp get-starknet and I would love to hear feedback from wallet providers.

**Comment by broody** [comment]
Hi @fracek! we're working on integrating the new WalletAccount APIs in controller, from what I understand, this should just work with get-starknet right?

**Comment by fracek** [comment]
Yes, if you implement with the wallet account api it should be easy to make it work.

**Comment by Cybermaxi7** [comment]
I’d like to work on this issue to integrate the Cartridge Controller into the default connectors. I have experience with API integrations, modular system design, and working with various connectors in web applications.

**Comment by OscarDom1** [comment]
I’m a Software Engineer/Developer with experience working on StarkNet and writing Cairo. I'd like to request the addition of Cartridge Controller to the default connectors. If assigned, my estimated turnaround time is 72 hours.
